src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -73,6 +75,11 @@ function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingE
                     }
                     setOnboardingCompanySize(selectedCompanySize);
 
+                    // Redirect is disabled on desktop
+                    if (getPlatform() !== CONST.PLATFORM.DESKTOP) {
+                        switchToOldDotOnNonMicroCompanySize(selectedCompanySize);
+                    }
```

Comment:
@mateuuszzzzz if we could replace the platform condition with a followup PR that would be great. Could we provide platform specific `switchToOldDotOnNonMicroCompanySize` functions which noops on Desktop?

Reply:
This is not essential for the PR though.

Reply:
Absolutely! I had a discussion about this today as it was hard to follow right patterns in this `onPress` callback and finally end up with this approach. I can open follow-up PR on Monday 😅